USAGE_android.md

```diff
@@ -102,8 +213,135 @@ It may also be set as a post attach command in the project configuration:
 * Enter the `process handle SIGSEGV -n true -p true -s false` command
 * Click "Apply"
 
+
+### To Root Or Not To Root
+
+Rooting the Android device for capture is not required if you are attempting
+to capture an application you can build in Debug mode and associate it with the
+GfxReconstruct layer during the build.
+However, if you can not do that, your only option is to attempt to capture
+the image on a rooted device.
+Instructions for rooting Android devices are available on the internet, but we
+won't reference any instructions here.
+
+If not building the application with the layer, several additional steps are
+required once the application is installed on a rooted Android device:
+
+#### Start ADB Shell With Admin Privileges
+
+```bash
+adb shell
+su
+```
+
+#### Locate the Application Data Directory
+
+```bash
+find /data/app -name *${Application Full Name}*
+```
+
+**NOTE:** Replace `${Application Full Name}` with the full Android name of the
+application being captured, for example "com.khronos.vulkan_samples" is the full
+name of the Vulkan Samples.
+You can find out the application full name by watch Logcat output if you don't
+already know it.
+
+#### Find the Appropriate Capture Layer .so
+
+By default, the capture layer's shared object file will be written to:
+
+```bash
+./android/layer/build/intermediates/cmake/${Build Type}/obj/${Build Target}/libVkLayer_gfxreconstruct.so
+```
+
+`${Build Type}` is the build type used when compiling the layer: "debug" or
+"release".
+
+`${Build Target}` is the platform target: "arm64-v8a", "armeabi-v7a", etc.
+
+#### Copy the Capture Layer Onto the Device
+
+The data folder returned by step 1 above can be used to store the layer, but
+it must be plaed in the appropriate library path.
+However, you can't copy it directly, so you need to first push the file into
+either the `/sdcard` or `/storage/emulated/0/Download` directory with an adb
+command:
+
+```bash
+adb push \
+    ./android/layer/build/intermediates/cmake/${Build Type}/obj/${Build Target}/libVkLayer_gfxreconstruct.so \
+    /sdcard
+```
+
+For example, for a debug build of the ARM64 version of the library:
+
+```bash
+adb push \
+    ./android/layer/build/intermediates/cmake/debug/obj/arm64-v8a/libVkLayer_gfxreconstruct.so \
+    /sdcard
+```
+
+Then move it to the application folder using the adb shell with Admin authority
+from step 1:
```

Comment:
I don't know what "step 1" is, but you could say "adb root shell" and just leave it at that.